bin/bootstrap_practice_exercise.exs

```diff
@@ -0,0 +1,258 @@
+# Generate all files required for a practice exercise.
+# File content is filled as much as possible, but some, including tests, need manual input.
+#
+# Run the following command from the root of the repo:
+# $ elixir bin/boostrap_practice_exercise.exs complex-numbers
+# Pass the name of the exercise (e. g., "complex-numbers") as an argument
+
+defmodule Generate do
+  def to_snake_case(lower_camel_case) do
+    lower_camel_case
+    |> String.split(~r/(?=[A-Z\d])/)
+    |> Enum.map_join("_", &String.downcase/1)
+  end
+
+  def get_properties(%{"cases" => cases}) when is_list(cases) do
+    Enum.map(cases, &Generate.get_properties/1)
+    |> Enum.concat()
+    |> Enum.uniq()
+  end
+
+  def get_properties(%{"property" => property, "input" => %{} = input}),
+    do: [
+      {property,
+       Enum.map(input, fn
+         {var, %{} = val} -> {var, Enum.map(val, fn {v, _} -> v end)}
+         {var, _} -> {var, nil}
+       end)}
+    ]
+
+  def get_properties(%{"property" => property}), do: [{property, "input"}]
+  def get_properties(_data), do: []
+
+  def print_property({property, variables}) do
+    """
+     @doc \"\"\"
+     insert function description here
+     \"\"\"
+     @spec #{property}(#{Enum.map_join(variables, ", ", fn
+      {var, nil} -> "#{var} :: String.t()"
+      {var, sub_vars} -> "#{var} :: %{#{Enum.map_join(sub_vars, " : String.t(), ", fn v -> v end)} : String.t()}"
+    end)}) :: {:ok, String.t()} | {:error, String.t()}
+     def #{property}(#{Enum.map_join(variables, ", ", fn {var, _} -> var end)}) do
+     end
+    """
```

Comment:
> Is it better to make them obviously wrong?

Maybe it's better to somehow mark them as requiring manual changes? Maybe something like this? (and similar in other places that also have most-likely-wrong defaults)
```suggestion
    """
     @doc \"\"\"
     TODO: insert function description here
     \"\"\"
     @spec #{property}(#{Enum.map_join(variables, ", ", fn
      {var, nil} -> "#{var} :: TODO.t()"
      {var, sub_vars} -> "#{var} :: %{#{Enum.map_join(sub_vars, " : TODO.t(), ", fn v -> v end)} : TODO.t()}"
    end)}) :: TODO.t()
     def #{property}(#{Enum.map_join(variables, ", ", fn {var, _} -> var end)}) do
     end
    """
```